src/worker.cc

```diff
@@ -142,6 +142,12 @@ void Worker::Stop() {
   }
 }
 
+void Worker::Sleep(uint64_t microsecond) {
+  // For other threads, don't allow to handle new connections during sleeping
```

Comment:
look like the lock is not necessarily, when in sleep, current worker won't accept new connection (what's this debug sleep mainly use for ?  online instance usually hava 8 workers, pause one worker and then check something about the current worker thread?)

Reply:
Actually, I want to block all threads to stop accepting new connections. In some redis tests, we may hope server don't process any request. As we know, In kvrocks, every thread is separate, but now i realize my way is wrong, i can set only one work thread to implement that. So it is unnecessary, thanks @karelrooted